### PR TITLE
Make ContainerInstance a standalone SDK

### PR DESCRIPTION
**Pull request by yolocs**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/containerinstance/resource-manager/readme.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+
+``` yaml $(python)
+python:
+  # override the default output folder
+  output-folder: $(output-folder)/python
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
```
Comment (lmazuel) [inline specification/containerinstance/resource-manager/readme.md]: Add namespace: `azure.mgmt.containerinstance`

I will override for now in my conf, but would be nice directly here

File: specification/containerinstance/resource-manager/readme.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+### Basic Information 
+These are the global settings for the ContainerInstance API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-08-preview
+```
+
+
+### Tag: package-container-instance-2017-08-preview
```
Comment (lmazuel) [inline specification/containerinstance/resource-manager/readme.md]: Blocking: this tag is incorrect and should be `package-2017-08-preview` in the three places used

File: specification/containerinstance/resource-manager/readme.md (hunk 1 above)
Comment (yolo3301) [inline specification/containerinstance/resource-manager/readme.md]: Added. Thanks!

**Review by lmazuel — CHANGES_REQUESTED** [review]
Incorrect tag

File: specification/containerinstance/resource-manager/readme.md (hunk 2 above)
Comment (yolo3301) [inline specification/containerinstance/resource-manager/readme.md]: Corrected.

**Comment by lmazuel** [comment]
Title is incorrect:
`"title": "Microsoft Container Instance Management Client"` 
should not contain `Microsoft`. This field is used to generate the client name, it's not just documentation.

I will override it for now in Python. @olydis for guidance.

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
I approved the tag fix, because it allows me to use the MD, but title is still to be fixed.

**Comment by yolo3301** [comment]
@lmazuel Title updated

**Comment by lmazuel** [comment]
Python PR: https://github.com/Azure/azure-sdk-for-python/pull/1330
I'm starting the Python review to see if there is more to change at the Swagger level

**Comment by lmazuel** [comment]
Need guidance to confirm `list` best practices ( @olydis @fearthecowboy @sergey-shandar @veronicagg @amarzavery please :)), but in my understanding:
- List by subscription should be called "list" (not "list_all" as currently)
- List by RG should be called "list_by_resource_group" (not "list" as currently)

**Comment by olydis** [comment]
@lmazuel Absolutely correct, thanks for catching that. Furthermore:
- shouldn't `tail` be an integer instead? https://github.com/yolo3301/azure-rest-api-specs/blob/dd1b3f3d090c21d4e835de87c038b309f2af392e/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json#L278
- I heard that your RP doesn't support updates. But then there is `ContainerGroups_CreateOrUpdate`. Which is correct?

**Comment by yolo3301** [comment]
@olydis Update is going to be supported in the near future in this API.

File: specification/compute/resource-manager/readme.md [hunk 3]
```diff
@@ -61,16 +61,6 @@ input-file:
 - Microsoft.Compute/2017-03-30/runCommands.json
 ```
 
-### Tag: package-container-instance-2017-08-preview
-
```
Comment (olydis) [inline specification/compute/resource-manager/readme.md]: thanks for removing 👍

**Review by olydis — APPROVED** [review]

**Comment by olydis** [comment]
synced with @dsgouda, awaiting his signoff

**Comment by yolo3301** [comment]
@olydis @lmazuel @dsgouda Thanks guys!

**Comment by dsgouda** [comment]
Signing off, thanks @olydis , @lmazuel

**Pull request merged**